code/datums/uplink_items/uplink_nuclear.dm

```diff
@@ -517,6 +509,14 @@
 
 // IMPLANTS
 
+/datum/uplink_item/implants/krav_implant
+	name = "Krav Maga Implant"
+	desc = "A biochip that teach you Krav Maga when implanted, great as a cheap backup weapon. Warning, the biochip will override any other fighting styles such as CQC."
```

Comment:
```suggestion
	desc = "A biochip that teaches you Krav Maga when implanted, great as a cheap backup weapon. Warning, the biochip will override any other fighting styles such as CQC."
```
Also, can you double check if it still overrides CQC with the new martial arts weight system Wilk made?

Reply:
Done. Added to desc to state that it will only override other martial arts when ACTIVE as its not passively on like the gloves are - you have to click a button to turn it on/off.